test/features/SIC AP/CBS Login.feature

```diff
@@ -0,0 +1,25 @@
+
+@auth @CBS @login @nosso
```

Comment:
```suggestion
@auth @CBS @login @no-sso
```

Reply:
Probably better to use this convention, since there will eventually be many tags.

Reply:
This is perfect for me